app/pb_kits/playbook/pb_time/docs/_time_sizes.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Time from '../_time.jsx'
+
+const TimeSizes = () => {
+  return (
+    <div>
```

Comment:
```suggestion
    <Fragment>
```

I think that if we use `Fragment` it is easier for developers to see where the implementation of the kit can be copied from.

Reply:
Sounds good to me. I've actually wondered about using these for a while as containers in docs.